### ISSUE TITLE
Unable to export a document with an undefined language identifier

### ISSUE THREAD

**Issue by fxha** [marktext/marktext#591]
### Description

Documents with code blocks cannot  be exported as HTML or PDF.

- [x] Can you reproduce the issue? 

### Steps to reproduce

1. Create a document with at least one code block without language identifier
2. Try to export the document as HTML or PDF
3. See error log

**Stack Trace**

Commit 02fb2f0:

```
vue.esm.js?a026:591 [Vue warn]: Error in event handler for "export": "TypeError: Cannot read property 'rest' of undefined
Please report this to https://github.com/marktext/marktext/issues."

(found in <Root>)
warn @ vue.esm.js?a026:591
logError @ vue.esm.js?a026:1737
globalHandleError @ vue.esm.js?a026:1732
handleError @ vue.esm.js?a026:1721
Vue.$emit @ vue.esm.js?a026:2540
LISTEN_FOR_EXPORT_PRINT.electron__WEBPACK_IMPORTED_MODULE_1__.ipcRenderer.on @ editor.js?6d24:471
emit @ events.js:182

vue.esm.js?a026:1741 TypeError: Cannot read property 'rest' of undefined
Please report this to https://github.com/marktext/marktext/issues.
    at Object.tokenize (/home/felix/projects/nodejs/marktext/node_modules/prismjs2/prism.js:426)
    at Object.highlight (/home/felix/projects/nodejs/marktext/node_modules/prismjs2/prism.js:290)
    at Object.highlight (exportHtml.js?9e71:24)
    at Renderer.code (renderer.js?b06a:42)
    at Parser.tok (parser.js?53e1:92)
    at Parser.parse (parser.js?53e1:26)
    at marked (index.js?9575:13)
    at ExportHtml.renderHtml (exportHtml.js?9e71:22)
    at ExportHtml.generate (exportHtml.js?9e71:43)
    at Muya.exportStyledHTML (index.js?64a7:86)
```

**exportHtml.js?9e71:24:**

```js
class ExportHtml {
// ...
  renderHtml () {
    return marked(this.markdown, {
      highlight (code, lang) {
        return Prism.highlight(code, Prism.languages[lang], lang) // <-- here
      },
      emojiRenderer (emoji) {
// ...
}
```

### Versions

- Mark Text: v0.13.65 / master
- Operating system: all

**Comment by fxha** [comment]
> I didn't see the error log when I follow your reproduce steps

OK, it have to be a code block without language identifier.

![mt_code_block_export](https://user-images.githubusercontent.com/22716132/49445339-fbd3fe00-f7d1-11e8-8b27-b164487b750e.gif)

> But I found that the code block background color changed after I export PDF file

That's right, another bug.

I also noticed that the syntax highlighting is a little bit strange:

![mt_code_block_syntax_highlighting](https://user-images.githubusercontent.com/22716132/49445619-9fbda980-f7d2-11e8-8f8f-a35a347852c7.gif)

**Comment by fxha** [comment]
> But I found that the code block background color changed after I export PDF file.

Should be fixed by #594.
